tensorflow/compiler/xla/service/gpu/matmul_utils.cc

```diff
@@ -402,18 +402,20 @@ StatusOr<bool> CanFoldTransposeOperandIntoDot(const HloInstruction& dot,
 }
 
 StatusOr<se::blas::ComputationType> GetBlasComputationType(
-    PrimitiveType dtype, int64_t compute_precision) {
+    PrimitiveType lhs_dtype, PrimitiveType dtype, int64_t compute_precision) {
```

Comment:
Rename `dtype` to `output_dtype`. And do the same in declaration in the header file.

(Sorry for not mentioning this until now).